modules/dnn/src/dnn.cpp

```diff
@@ -1269,6 +1273,8 @@ struct Net::Impl
             allocateLayer(lid, layersShapes);
         }
 
+        layersTimings.resize(lastLayerId + 1, 0);
```

Comment:
We have some bug with `lastLayerId` value - it initializes with 1 but must be initialized by 0 (because initially we have single input layer with id == 0). Please, replace `lastLayerId = 1;` to `lastLayerId = 0;` at `Net::Impl` constructor.